SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1284,6 +1289,19 @@ void USpatialReceiver::ProcessQueuedResolvedObjects()
 	ResolvedObjectQueue.Empty();
 }
 
+void USpatialReceiver::ProcessQueuedActorRPCsOnEntityCreation(AActor* Actor, SpatialGDK::RPCsOnEntityCreation* QueuedRPCs)
```

Comment:
```suggestion
void USpatialReceiver::ProcessQueuedActorRPCsOnEntityCreation(AActor* Actor, const SpatialGDK::RPCsOnEntityCreation& QueuedRPCs)
```

Reply:
Cannot be const as it contains RPCPayload that is eventually sent as a non-const argument to UE4 BitWriter